oshi-core/src/main/java/oshi/driver/windows/registry/HkeyPerformanceDataUtil.java

```diff
@@ -272,7 +273,7 @@ private static Map<String, Integer> mapCounterIndicesFromRegistry() {
             String[] counterText = Advapi32Util.registryGetStringArray(WinReg.HKEY_LOCAL_MACHINE, HKEY_PERFORMANCE_TEXT,
                     COUNTER);
             for (int i = 1; i < counterText.length; i += 2) {
-                indexMap.putIfAbsent(counterText[i], Integer.parseInt(counterText[i - 1]));
+                indexMap.putIfAbsent(counterText[i], ParseUtil.parseLastInt(counterText[i - 1], 0));
```

Comment:
I think we can leave this one as-is and just add a `@SuppressForbidden` annotation.  We want to throw an exception here (it's handled on line 282) rather than quietly stick a 0 in the map which would possibly lead to other bugs when that 0 was used.

Reply:
Done